1091. Shortest Path in Binary Matrix/solution.py

```diff
@@ -0,0 +1,42 @@
+class Solution(object):
+	# Given an n x n binary matrix grid, we return the shortest clear path in 
+	# the matrix, or -1 for no clear path
+
+	# A clear path is a path from the top-left cell to the bottom right cell
+	# such that:
+
+	 	# 1. all visited cells are 0
+	 	# 2. all adjacent cells are different and share an edge or corner
+
+
+	def findAdjacent(matrix, start_row, start_col):
+		# create method to find surrounding cells that are clear path, given a  
+		# matrix and a current location
+
+			# define directions as tuples (-1, 0), (1, 0), (0, -1), (0, 1), 
+			# (-1, -1), (-1, 1), (1, -1), (1, 1)
+
+			# for the given location, check if there are any 0's in the 8  
+			# surrounding spots, check if any of these are out of bounds
```

Comment:
nit: I usually use full sentences for comments.

Reply:
Agreed that this comment is a little unclear. I will adopt the practice of usually using full sentences.